cmd/dotmesh-server/http.go

```diff
@@ -87,8 +87,9 @@ func (state *InMemoryState) runServer() {
 			),
 		).Methods("POST")
 
-		// display diff since the last commit
+		// display diff since the last commit, or the latest named commit
 		router.Handle("/diff/{namespace}:{name}", Instrument(state)(NewAuthHandler(NewDiffHandler(state), state.userManager))).Methods("GET")
+		router.Handle("/diff/{namespace}:{name}/{snapshotID}", Instrument(state)(NewAuthHandler(NewDiffHandler(state), state.userManager))).Methods("GET")
```

Comment:
this also has to be (probably) added below, where we go into else {...

Reply:
burning the zipkin stuff in a fire 🔥